apps/website/screens/components/data-grid/code/examples/hierarchicalSelectable.ts

```diff
@@ -1,8 +1,8 @@
 import { DxcDataGrid, DxcInset } from "@dxc-technology/halstack-react";
-import { useState } from "react";
+import { useMemo, useState } from "react";
 
 const code = `() => {
-  const columns = [
+  const columns = useMemo(() => [
```

Comment:
Instead of useMemo I think useState without the setter is more correct for this case.